### ISSUE TITLE
bug - daytona swallows errors

### ISSUE THREAD

**Issue by geekflyer** [cruise-automation/daytona#48]
Generally daytona seems to swallow a bunch of different errors when fetching secrets fails.
Here is an example I ran into recently, but there are other instances I ran into this:

use these environment variables:
```
        - name: VAULT_SECRET_DATABASE_URL
          value: my/valid/vault/path/DATABASE_URL
        - name: DAYTONA_SECRET_DESTINATION_DATABASE_URL
          value: /secrets/folder_that_does_not_yet_exist/DATABASE_URL
```

In this example I specified a destination folder that does not yet exist.
What I would expect is that daytona either:
- creates the target directory
- errors out with a message that the target directory does not exist and errors with a non-zero exit-code.

What daytona really prints is this:
```
DAYTONA - 2020/08/14 08:16:45 Starting secret fetch
DAYTONA - 2020/08/14 08:16:45 Certificate or private key output path is empty, will not attempt to get certificate
```
and exits with a 0 exit code.

So TLDR it simply skips that secret above without any indication/error which makes it very difficult to catch and debug these configuration errors.

I'm not sure what the last message "Certificate or private key output path is empty" means but I seem to get this error messages also in cases where the secret fetch actually works.

**Comment by broamski** [comment]
Verbose, per-secret processing log entries were added in [v1.1.5](https://github.com/cruise-automation/daytona/releases/tag/v1.1.5). Thanks!
